package.json

```diff
@@ -92,8 +92,5 @@
     ],
     "jasmine-core": "3.10.1",
     "iltorb": "npm:@stdlib/utils-noop@0.0.10"
-  },
-  "dependencies": {
-    "iltorb": "npm:@stdlib/utils-noop@0.0.10"
```

Comment:
Not sure why this was added, but it's not necessary.

It is annoying, though, that this `resolutions` trick doesn't play well with `yarn upgrade:interactive --latest`. I had to delete the `yarn.lock` file to get it to work.